tests/WorkOS/ClientTest.php

```diff
@@ -85,7 +85,7 @@ public function testClientThrowsRequestExceptionsWithBadMessage($statusCode, $ex
         try {
             Client::request(Client::METHOD_GET, $path);
         } catch (Exception\BaseRequestException $e) {
-            $this->assertEquals($e->getMessage(), "");
+            $this->assertEquals($e->getMessage(), $result);
```

Comment:
Could we add assertions for the other response possibilities? It seems we are only testing the `$e->getMessage()`. Can we check if the content of `$e->$responseError`, `$e->$responseErrors`, `$e->$responseCode`, `$e->$responseErrorDescription` is right?

Reply:
Thanks @jonatascastro12 - I added new unit tests for these, as well. Great call. I bundled code & message, error & error description, and then left errors as it's own unit test, as I believe this is how the payloads are returned from the API.